Automattic/jetpack
# i18n: Update i18n-calypso package to latest version.

#### Changes proposed in this Pull Request:

@see https://github.com/Automattic/i18n-calypso/releases/tag/3.0.0

This release allows us to use translations brought over from Calypso via the SDK:
https://github.com/Automattic/wp-calypso/blob/master/docs/sdk.md

#### Testing instructions:

* Set some other site language in Settings -> General
* Make sure translations still work for Jetpack React
  * To test, go to `/wp-admin/admin.php?page=jetpack#/settings` and verify strings appear translated in the UI
* Make sure translations work for Gutenberg i18n
  * Make sure you have the latest Gutenberg installed.
  * Start a new post `/wp-admin/post-new.php`
  * Run `wp.i18n.__( 'Settings', 'jetpack' )` in your browser console (making sure that this string has a translation for the selected language).
  * Verify a translated value is returned.

I was able to run `yarn build-languages` with no apparent issues after updating. However, it is worth noting that the new strings part of `/_inc/blocks/editor.js` in my local install were not added to `/_inc/jetpack-strings.php`, so we'll still have to figure this out for translations to work.


#### Proposed changelog entry for your changes:
None
